### PR TITLE
chore: disable comments unless user is signed in

### LINKED ISSUE #501: todo: disable commenting if user is not signed in
If the user is not signed in:
- The comment input and button must be disabled
- The comment button should maybe change to "Sign in to comment" or similar

### PR DESCRIPTION
**Pull request by mbhrznr**
closes #501

### CODE REVIEW

File: routes/items/[id].tsx [hunk 1]
```diff
@@ -7,17 +7,20 @@ import Head from "@/components/Head.tsx";
 import type { State } from "@/middleware/session.ts";
 import CommentsList from "@/islands/CommentsList.tsx";
 
-function CommentInput(props: { itemId: string }) {
+function CommentInput(props: { isLoggedIn: boolean; itemId: string }) {
```
Comment (iuioiua) [inline routes/items/[id].tsx:10]:
```suggestion
function CommentInput(props: { isSignedIn: boolean; itemId: string }) {
```
Nit: for consistency. Ditto to other instances of the variable.

**Review by iuioiua — CHANGES_REQUESTED** [review]
Looking good! Just one nit

File: routes/items/[id].tsx (hunk 1 above)
Comment (mbhrznr) [reply routes/items/[id].tsx]: changed as proposed!

**Review by iuioiua — APPROVED** [review]
LGTM! Thanks Michael.

**Pull request merged**